extensions/systemd/index.js

```diff
@@ -17,7 +17,15 @@ class SystemdExtension extends cli.Extension {
     }
 
     _setup(argv, ctx, task) {
-        const uid = getUid(ctx.instance.dir);
+        let uid;
+
+        // getUid returns either the uid or null, but can also throw an error
```

Comment:
`getUid` was always able to throw an error, which we didn't handle anywhere. Therefore this refactor (same [here](https://github.com/TryGhost/Ghost-CLI/pull/637/files#diff-1edc5bfa2e72eb7d9a548201e7eb9674R104)).